### PR TITLE
Add ingest.sh script.

### PR DESCRIPTION
**Pull request by pantierra**
Data Management (data ingestion in a first step) is a common requirement for several of our `eoapi-k8s` deployments and projects.

From my perspective we should think about data on eoAPI as "content" and not so much as "configuration". While configuration should be in code and part of the reproducible deployment, content should me managable by an admin user. At some point we probably want to think about building some auth-supported API to allow management, eventually even connecting it to some user interface.

This PR consists in a first simple step to add very basic ingestion as a bash script `ingest.sh` and the related `make ingest` command, it:

* looks for a kubernetes the eoAPI `raster` pod in the namespaces `eoapi` and `default`
* it installs `pypgstac` on it
* it uploads `collection.json` and `items.json` from the current directory (can be overridden with the environment variables `EOAPI_COLLECTIONS_FILE` and `EOAPI_ITEMS_FILE`.
* it `pypgstac` loads the collection and items into the pySTAC postgresql database.

Related to https://github.com/EOEPCA/data-access/issues/107

fyi, @j08lue, @emmanuelmathot, @zacharyDez

### CODE REVIEW

**Comment by pantierra** [comment]
I dived a bit more into this topic. The best way to have an API seems to be relying on the `stac-fastapi-pgstac` endpoints implementing the [STAC API Transactions extension](https://github.com/stac-api-extensions/transaction). Currently there is no simple way to protect these endpoints. Currently people doing this either using an API Gateway, or writing a custom `stac-fastapi-pgstac`.

Anyway this `ingest.sh` script and command will ease up the ingestion in the meantime.

File: docs/manage-data.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Data management
+
+For now, data management in eoAPI is rudimentary and mostly a manual process on the components of the stack.
+However, we aim to improve stepwise and provide more automation in the future. This documentation will be updated as we make progress.
```
Comment (j08lue) [inline docs/manage-data.md:4]: I would perhaps only describe the current situation, not possible future ones:

```suggestion
eoAPI-k8s provides a basic data ingestion process that consist of manual operations on the components of the stack.
```

**Review by j08lue — COMMENTED** [review]
I will leave running the script to others.

It looks nicely laid out and documented and is providing at least a basic mechanism for data ingestion, which is a battery we have to include, IMO.

File: docs/manage-data.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Data management
+
+eoAPI-k8s provides a basic data ingestion process that consist of manual operations on the components of the stack.
+
+# Load data
+
+## Preshipped bash script
+
+Execute `make ingest` to load data into the eoAPI service.
```
Comment (ciaransweet) [inline docs/manage-data.md:9]: I think this should explicitly call out that the `make ingest` target _expects_ two files at the root of the repo. (To save having to dive into a bash script/errors to find that out

File: docs/manage-data.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+### Import data
+
+On the pod or server running the raster eoAPI service, you can use the following commands to load the data:
+
+```bash
+#!/bin/bash
+apt update -y && apt install python3 python3-pip -y && pip install pypgstac[psycopg]';
+pypgstac pgready --dsn $PGADMIN_URI
+pypgstac load collections /tmp/collection.json --dsn $PGADMIN_URI --method insert_ignore
```
Comment (ciaransweet) [inline docs/manage-data.md:28]: Similarly, given this needs data mounted on `/tmp` we probably want to hand hold people into how to do that.

File: ingest.sh [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    echo "Failed to install packages."
+    exit 1
+fi
+
+# Copy files to pod
+echo "Copying files to pod..."
+echo "Using collections file: $EOAPI_COLLECTIONS_FILE"
+echo "Using items file: $EOAPI_ITEMS_FILE"
+kubectl cp "$EOAPI_COLLECTIONS_FILE" "$FOUND_NAMESPACE/$EOAPI_POD_RASTER":/tmp/collections.json
+kubectl cp "$EOAPI_ITEMS_FILE" "$FOUND_NAMESPACE/$EOAPI_POD_RASTER":/tmp/items.json
```
Comment (ciaransweet) [inline ingest.sh:60]: I.E just put these 2 lines in your prerequisites section!

**Review by ciaransweet — CHANGES_REQUESTED** [review]
Overall looks great to me, a real step in to making this easier

Just a few comments that I think will waterproof this

File: docs/manage-data.md (hunk 2 above)
Comment (pantierra) [reply docs/manage-data.md]: Yes, indeed, thanks for raising this. I adjusted the docs.

File: docs/manage-data.md (hunk 3 above)
Comment (pantierra) [reply docs/manage-data.md]: Yes, indeed, thanks for raising this. I adjusted the docs.

File: ingest.sh (hunk 4 above)
Comment (pantierra) [reply ingest.sh:60]: Yes, indeed, thanks for raising this. I adjusted the docs.

**Review by ciaransweet — APPROVED** [review]
LGTM!

**Comment by pantierra** [comment]
Many thanks for your reviews! :heart:

**Pull request merged**